amti/actions/review.py

```diff
@@ -3,6 +3,7 @@
 import json
 import logging
 import os
+import sys
```

Comment:
The simplifications suggested by my comment on amti/actions/review.py#L178-L183 would mean that this import can also be deleted.

Reply:
Done